src/registrar/admin.py

```diff
@@ -1240,14 +1240,14 @@ class DraftDomainAdmin(ListHeaderAdmin):
 
 
 class VeryImportantPersonAdmin(ListHeaderAdmin):
-    list_display = ("email", "user", "truncated_notes", "created_at")
+    list_display = ("email", "requestor", "truncated_notes", "created_at")
```

Comment:
(off topic) Ah the old british vs american spelling debate, color vs colour, I see?